.npmignore

```diff
@@ -1,2 +1,3 @@
 coverage
 .nyc_output
+features
```

Comment:
Should this also include entries in the .gitignore? i.e. maze_output, others?

Reply:
I believe .npmignore inherited rules from .gitignore (so your only need to specify the differences) but I will confirm.